components/projects/projects.sass

```diff
@@ -0,0 +1,12 @@
+.projects
+  height: 65vh
+  display: flex
+  flex-direction: column
+  align-items: center
+  justify-content: space-evenly
+
+.projects__header
+  font-size: 3rem
```

Comment:
Are we using the same font size and weight on other headers?
If so, let's move these values to variables in a SASS config and setup a mixin such as "section-header" that we can re-use throughout the site.